libcodechecker/analyze/analyzers/config_handler.py

```diff
@@ -65,27 +65,15 @@ def add_checker(self, checker_name, enabled, description):
         """
         self.__available_checkers[checker_name] = (enabled, description)
 
-    def enable_checker(self, checker_name, description=None):
+    def set_checker_enabled(self, checker_name, enabled):
```

Comment:
We can set a default value for `enabled` parameter (e.g.: `True`). This way no need to write out this value explicitly on function calls when the value is `True`.